### PR TITLE
Added Cludo to products

### PR DESCRIPTION
**Pull request by PhilipAnd**

### CODE REVIEW

File: _data/products/cludo.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "name": "Cludo Search",
+    "url": "https://www.cludo.com/en",
+    "logo_url": "https://www.cludo.com/img/Cludo-full-color-dark-h40.png",
+    "top_keywords": [
+        "enterprise search",
+        "search",
+        "search solution",
+        "website search",
+        "intranet search",
+        "subsite search"
```
Comment (stroupaloop) [inline _data/products/cludo.json]: please only select 5 keywords

**Comment by stroupaloop** [comment]
hi @Phatzor made a small note on only using five keywords. Besides that, it looks good to go!

**Comment by PhilipAnd** [comment]
@stroupaloop Sorry didn't see the text inside the JSON example, describing that there can only be 5 keywords. Maybe an idea would be to highlight it so you won't get too much work in the future :)

**Comment by stroupaloop** [comment]
Thanks @Phatzor agreed, will review and update the README

**Pull request merged**